src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/DefaultComesLastCheck.java

```diff
@@ -214,13 +214,8 @@ else if (Objects.nonNull(findNextSibling(defaultGroupAST,
      */
     private static DetailAST findNextSibling(DetailAST ast, int tokenType) {
         DetailAST token = null;
-        DetailAST node = ast.getNextSibling();
-        while (node != null) {
-            if (node.getType() == tokenType) {
-                token = node;
-                break;
-            }
-            node = node.getNextSibling();
+        if (ast.getNextSibling() != null && ast.getNextSibling().getType() == tokenType) {
+            token = ast;
```

Comment:
Omg.
Please rename method to be `boolean isNextSiblingOf(DetailAST ast, int tokenType)`
And refactor all usages to be simplified.

Reply:
Done.